docs/user-guide/persistent-volumes/index.md

```diff
@@ -222,6 +222,51 @@ A volume will be in one of the following phases:
 
 The CLI will show the name of the PVC bound to the PV.
 
+### Mount Options
+
+Kubernetes administrator can specify additional mount option to be used when a Persistent Volume is
```

Comment:
Who is "Kubernetes administrator" in this case? The reader?

If the reader, address as "you." As in, "You can specify an additional mount option for the system to use when a Persistent Volume is mounted on a node."